src/v2/view-builder/views/IdentifierView.js

```diff
@@ -61,6 +63,17 @@ const Body = BaseForm.extend({
     // add external idps buttons
     const idpButtons = createIdpButtons(this.options.appState.get('remediations'));
     if (Array.isArray(idpButtons) && idpButtons.length) {
+
+      // OKTA-396776: Add the forgot password link before the buttons for multiple IDPs
```

Comment:
No need to tag Jira, if it's not a TODO item.

Reply:
done